app/Helpers/database/game.php

```diff
@@ -108,6 +108,15 @@ function getGameMetadata(
         $gameDataOut['ParentGameID'] = $parentGameId;
         $gameDataOut['NumDistinctPlayers'] = $gameMetrics['NumDistinctPlayers'] ?? 0;
 
+        // Handle the rare situation where there is a parent game ID that doesn't have a set.
+        // We should only fall into this block if there's a subset linked to a parent game ID
+        // that actually has no players. In this case, use the subset's true player count.
+        if ($parentGameId && $gameDataOut['NumDistinctPlayers'] === 0) {
+            $query = "SELECT players_total AS NumDistinctPlayers FROM GameData WHERE ID=" . $gameID;
+            $gameMetrics = legacyDbFetch($query);
+            $gameDataOut['NumDistinctPlayers'] = $gameMetrics['NumDistinctPlayers'] ?? 0;
+        }
```

Comment:
Is this block actually necessary? The UpdateGameMetrics code should set the subset's player_total to always be non-zero.

Reply:
You're right - this isn't necessary. The UpdateGameMetrics change should naturally resolve the issue. Not sure why I decided to add this and didn't make that connection. Anyways, I've reverted these _game.php_ changes in latest.